models/project/issue.go

```diff
@@ -100,6 +146,27 @@ func MoveIssuesOnProjectColumn(ctx context.Context, column *Column, sortedIssueI
 	})
 }
 
+func MoveIssueToColumnTail(ctx context.Context, issue *ProjectIssue, toColumn *Column) error {
+	ctx, committer, err := db.TxContext(ctx)
+	if err != nil {
+		return err
+	}
+	defer committer.Close()
+
+	num, err := toColumn.NumIssues(ctx)
```

Comment:
This is not right. If you update two columns, then they will have the same `sorting`.

Reply:
but this function only can update one column.

Reply:
> I mean one by one.

I think it okay, becaue when call it one by one, NumIssues also will be canged one by one. else which `sorting` should be seted in this usage? zero?

Reply:
looks in `moveIssuesToAnotherColumn()`, it use `max()` sql function to get new `sorting`, I have use same logic in https://github.com/go-gitea/gitea/pull/30617/commits/a824db0d8b986ac88926bcf656e431ed77a3ae5d, how about this?